### ISSUE TITLE
Error in running of phiWB Wilson coefficient

### ISSUE THREAD

**Issue by peterstangl** [wilson-eft/wilson#7]
When trying to run Wilson coefficients containing a non-zero value of `phiWB`, I receive the following error I am not sure where it is coming from:

```python
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-17-3ce70e47007d> in <module>()
----> 1 mywilson.match_run(1e2, 'SMEFT', 'Warsaw')

~/.local/lib/python3.6/site-packages/wilson/classes.py in match_run(self, scale, eft, basis, sectors)
     90         if self.wc.eft == 'SMEFT':
     91             if eft == 'SMEFT':
---> 92                 smeft = SMEFT(self.wc)
     93                 # if input and output EFT ist SMEFT, just run.
     94                 wc_out = smeft.run(scale)

~/.local/lib/python3.6/site-packages/wilson/run/smeft/classes.py in __init__(self, wc, get_smpar)
     34         self.C_in = None
     35         if wc is not None:
---> 36             self._set_initial_wcxf(wc, get_smpar=get_smpar)
     37 
     38     def _set_initial(self, C_in, scale_in):

~/.local/lib/python3.6/site-packages/wilson/run/smeft/classes.py in _set_initial_wcxf(self, wc, get_smpar)
     78             self.C_in.update(C)
     79         if get_smpar:
---> 80             self.C_in.update(self._get_sm_scale_in())
     81 
     82 

~/.local/lib/python3.6/site-packages/wilson/run/smeft/classes.py in _get_sm_scale_in(self, scale_sm)
    177         _smeft = SMEFT(self.wc, get_smpar=False)
    178         # Step 1: run the SM up, using the WCs at scale_input as (constant) estimate
--> 179         _smeft.C_in.update(self._run_sm_scale_in(self.C_in, scale_sm=scale_sm))
    180         # Step 2: run the WCs down in LL approximation
    181         C_out = _smeft._rgevolve_leadinglog(scale_sm)

~/.local/lib/python3.6/site-packages/wilson/run/smeft/classes.py in _run_sm_scale_in(self, C_out, scale_sm)
    153         C_in_sm = beta.C_array2dict(np.zeros(9999))
    154         # set the SM parameters to the values obtained from smpar.smeftpar
--> 155         C_SM = smpar.smeftpar(scale_sm, C_out, basis='Warsaw')
    156         SM_keys = set(smeftutil.SM_keys)  # to speed up lookup
    157         C_SM = {k: v for k, v in C_SM.items() if k in SM_keys}

~/.local/lib/python3.6/site-packages/wilson/run/smeft/smpar.py in smeftpar(scale, C, basis)
    106     g = gbar * (1 - C['phiW'] * (v**2))
    107     ebar = sqrt(4 * pi * p['alpha_e'])
--> 108     gp = get_gpbar(ebar, gbar, v, C)
    109     c = {}
    110     c['m2'] = m2

~/.local/lib/python3.6/site-packages/wilson/run/smeft/smpar.py in get_gpbar(ebar, gbar, v, C)
     85                         (1 - eps * gb * gpb / (gb**2 + gpb**2)))
     86             return (ebar_calc - ebar).real
---> 87         gpbar = scipy.optimize.brentq(f0, 0, 3)
     88     return gpbar * (1 - C['phiB'] * (v**2))
     89 

~/.local/lib/python3.6/site-packages/scipy/optimize/zeros.py in brentq(f, a, b, args, xtol, rtol, maxiter, full_output, disp)
    517     if rtol < _rtol:
    518         raise ValueError("rtol too small (%g < %g)" % (rtol, _rtol))
--> 519     r = _zeros._brentq(f,a,b,xtol,rtol,maxiter,args,full_output,disp)
    520     return results_c(full_output, r)
    521 

ValueError: f(a) and f(b) must have different signs
```

This error is generated by the following code:
```python
from wilson import Wilson
mywilson = Wilson({'phiWB': 1}, 1e3, 'SMEFT', 'Warsaw')
mywilson.match_run(1e2, 'SMEFT', 'Warsaw')
```

**Comment by peterstangl** [comment]
Oh, yes, this makes totally sense, I forgot to fix the normalization...

However, I will keep the issue open as a reminder to perhaps catch the error and add a corresponding error message.

**Comment by peterstangl** [comment]
I have solved this issue in PR #11.

**Comment by DavidMStraub** [comment]
Great, thanks!
